pallet-chainlink-feed/src/lib.rs

```diff
@@ -73,14 +73,18 @@ pub mod pallet {
 		pub description: Vec<u8>,
 		/// The round initiation delay
 		pub restart_delay: RoundId,
-		/// The round oracles are currently reporting data for
+		/// The round oracles are currently reporting data for.
 		pub reporting_round: RoundId,
 		/// The id of the latest round
 		pub latest_round: RoundId,
 		/// The id of the first round that contains non-default data
 		pub first_valid_round: Option<RoundId>,
 		/// The amount of the oracles in this feed
 		pub oracle_count: u32,
+		/// Number of rounds to keep in storage for this feed.
+		pub pruning_window: RoundId,
```

Comment:
Shouldn't this be a number instead of RoundId?

Reply:
`RoundId` is actually only a type alias:
`type RoundId = u32;`
That is on par with https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/FluxAggregator.sol#L101
However, we could also make this a Config type in the feed pallet's `Config` trait instead? so that this can also be configured with u64, u128...

Reply:
Hm I see. Maybe for start just change this type to u32, since it refers to number of rounds instead of a certain id? In case for e.g in the future RoundId itself changes type to string or uuid or something. Does it make sense @mattsse